gradle/gradle-mvn-push.gradle

```diff
@@ -120,7 +120,9 @@ def androidArtifactTasks() {
         source = android.sourceSets.main.java.srcDirs
 
         classpath += project.files(android.getBootClasspath().join(File.pathSeparator))
-        exclude '**/internal/*'
+        android.libraryVariants.all { variant ->
+            classpath += variant.javaCompiler.classpath
+        }
```

Comment:
This `classpath` change solves most errors during javadoc except for errors around `R` and `BuildConfig` (generated files).  Fixing the last of these errors might take much deeper dive into gradle configs and options.